docs/SUMMARY.md

```diff
@@ -38,6 +44,20 @@
     * [Block Patterns](blocks/block-development/block-patterns.md)
     * [The nio Framework](blocks/block-development/framework.md)
     * [Mixins](blocks/block-development/mixins.md)
+* [User Interfaces](ui/README.md)
+  * [How to Build a UI with nio](ui/build-a-ui.md)
+
+---
+* Core Components
```

Comment:
I would rename this to `nio Core` probably, not all of these are technically components

Reply:
nio Core it is.